src/test/java/Domashka17.java

```diff
@@ -0,0 +1,42 @@
+import com.fasterxml.jackson.databind.ser.Serializers;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Domashka17 extends BaseTest {
+
+
+    @Test
+    public static void addSongToThePlaylist() {
+        BaseTest B = new BaseTest();
```

Comment:
Extension methods enable you to "add" methods to existing types without creating a new derived type, recompiling, or otherwise modifying the original type. 

Please, delete `BaseTest B = new BaseTest();`

For example 

```
  @Test
  public void addSongToThePlaylist() {
    setupBrowser();
    navigation("https://bbb.testpro.io/");
    enterEmail("valentineguts@gmail.com");
    enterPassword("te$t$tudent");
    submit();
    search("hozlinaCC0");
    // B.sleep();
    clickViewALl();
    clickFirstSong();
    AddTo();
    addToPlaylist();
    Assert.assertTrue(verifyNotification().contains("Added 1 song into"));

  }

}
```